development/tools/cmd/oomfinder/README.md

```diff
@@ -0,0 +1,17 @@
+# oomfinder
+
+## Overview
+
+oomfinder is a small tool designed to run in a Pod on each k8s worker node as a privileged container. It will check if docker or containerd is used and attach to its socket to listen for oom events. On oom event, oomfinder will print message to os stdout with following details.
```

Comment:
```suggestion
oomfinder is a small tool designed to run in a Pod on each k8s worker node as a privileged container. It will check if Docker or Containerd is used and attach to its socket to listen for oom events. If an oom event occurs, oomfinder will print a message to `os stdout` with the following details:
```